lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoilHeatingGas.rb

```diff
@@ -0,0 +1,129 @@
+class ASHRAE901PRM < Standard
+  # @!group BoilerHotWater
+
+  # find search criteria
+  #
+  # @param coil_heating_gas [OpenStudio::Model::CoilHeatingGas] coil heating gas object
+  # @param sys_type [String] HVAC system type
+  # @return [Hash] used for standards_lookup_table(model)
+  def coil_heating_gas_find_search_criteria(coil_heating_gas, sys_type)
+    # Define the criteria to find the furnace properties
+    # in the hvac standards data set.
+    search_criteria = {}
+    search_criteria['fuel_type'] = 'Gas'
+    if sys_type == 'PSZ_AC'
+      search_criteria['subcategory'] = 'Furnace'
+    elsif sys_type == 'Gas_Furnace'
+      search_criteria['subcategory'] = 'Unit Heater'
+    else
+      search_criteria['subcategory'] = 'Furnace'
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.CoilHeatingGas', "#{coil_heating_gas.name} is not for a PSZ-AC or Gas Furnace and cannot apply efficiency standard.")
+    end
+
+    return search_criteria
+  end
+
+  # find furnace capacity
+  #
+  # @param coil_heating_gas [OpenStudio::Model::CoilHeatingGas] coil heating gas object
+  # @return [Double] capacity in W to be used for find object
+  def coil_heating_gas_find_capacity(coil_heating_gas)
+    # Get the coil capacity
+    capacity_w = nil
+    if coil_heating_gas.nominalCapacity.is_initialized
+      capacity_w = coil_heating_gas.nominalCapacity.get
+    elsif coil_heating_gas.autosizedNominalCapacity.is_initialized
+      capacity_w = coil_heating_gas.autosizedNominalCapacity.get
+    else
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.CoilHeatingGas', "For #{coil_heating_gas.name} capacity is not available, cannot apply efficiency standard.")
+      successfully_set_all_properties = false
+      return successfully_set_all_properties
+    end
+
+    return capacity_w
+  end
+
+  # A helper method to convert from combustion efficiency to thermal efficiency
+  #
+  # @param combustion_eff [Double] Combustion efficiency (%)
+  # @return [Double] Thermal efficiency (%)
+  def combustion_eff_to_thermal_eff(combustion_eff)
+    return combustion_eff - 0.02
```

Comment:
What is the source for this formula? We should state it in the comments section.